datasrc/network.py

```diff
@@ -344,6 +345,11 @@
 		NetIntAny("m_Angle"),
 	]),
 
+	NetEventEx("Finish", "finish@netevent.ddnet.tw", [
```

Comment:
```suggestion
	NetEventEx("Finish:Common", "finish@netevent.ddnet.tw", [
```
Did that not work (while removing the `NetIntAny("m_X")` and `NetIntAny("m_Y")` below)?

If it doesn't work, tell me, then I'll try to get it fixed.